setup.py

```diff
@@ -63,6 +63,7 @@
         ],
         'console_scripts': [
             'labgrid-client = labgrid.remote.client:main',
+            'labgrid-coordinator = crossbar:run',
```

Comment:
Please remove this, not every labgrid installation will have crossbar available, since this is only required on the coordinator. If crossbar isn't available this falsely implies that the coordinator can be used.

Reply:
Done, removed.